src/tls/cockpit-certificate-helper.in

```diff
@@ -26,6 +26,65 @@ install_key() {
     fi
 }
 
+selfsign_sscg() {
+    sscg --quiet \
+        --lifetime "${DAYS}" \
+        --key-strength 2048 \
+        --cert-key-file "${KEYFILE}" \
+        --cert-file "${CERTFILE}" \
+        --ca-file "${CA_FILE}" \
+        --hostname "${HOSTNAME}" \
+        --organization "${MACHINE_ID}" \
+        --subject-alt-name localhost \
+        --subject-alt-name IP:127.0.0.1/255.255.255.255
```

Comment:
While we are at it, IPv6 ::1/128 too?

Reply:
This seems to be non-trivial.  It's not entirely clear what the correct format for IPv6 is: different sources have conflicting answers.  Let's look at it in a separate PR.

Reply:
Ack.